data/CMakeLists.txt

```diff
@@ -1,6 +1,8 @@
+find_package(PythonInterp REQUIRED)
+
 set(OPENCC_DICT_BIN opencc_dict)
-set(DICT_MERGE_BIN python ${CMAKE_CURRENT_SOURCE_DIR}/scripts/merge.py)
-set(DICT_REVERSE_BIN python ${CMAKE_CURRENT_SOURCE_DIR}/scripts/reverse.py)
+set(DICT_MERGE_BIN ${PYTHON_EXECUTABLE} ${CMAKE_CURRENT_SOURCE_DIR}/scripts/merge.py)
```

Comment:
Are quotes necessary, in case `PYTHON_EXECUTABLE` is a path containing space character?

Reply:
Yes, I will make an update.